app/coffee/RedisManager.coffee

```diff
@@ -24,16 +31,21 @@ module.exports = RedisManager =
 		logger.log project_id:project_id, doc_id:doc_id, version: version, hash:docHash, "putting doc in redis"
 		ranges = RedisManager._serializeRanges(ranges)
 		multi = rclient.multi()
-		multi.set keys.docLines(doc_id:doc_id), docLines
+		multi.eval setScript, 1, keys.docLines(doc_id:doc_id), docLines
```

Comment:
It doesn't cache for EVAL.
This turns out to be a good thing because we use EVAL inside a MULTI, which would create a problem if it failed with 'script does not exist'.  We would have to rewrite our MULTI if there was caching involved.  There are a few issues in the ioredis github about that.
Ioredis has its own defineCommand method which is meant to handle this.